index.js

```diff
@@ -0,0 +1,88 @@
+// Import 
+import { NovelCollection } from './modules/novelCollection.js';
+
+import { DateTime } from '../../node_modules/luxon/src/luxon.js';
+
+// some public consts
+
+// nav-btn-list
+// nav-btn-add
+// nav-btn-contact
+const novelListSection = document.getElementById('bookList');
+const addNewNovelSection = document.querySelector('.textfield-and-btn');
+const contactPageSection = document.querySelector('.contact-container');
+const pageHeaderSection = document.querySelector('.page-header');
+// novelCollection.display.style = 'none';
+// contactPageSection.display.style = 'none';
```

Comment:
[OPTIONAL] Please remove unwanted comments because it's your codebase messy if you want to check your previous code then  check your GitHub commit history :slightly_smiling_face: